datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -14,27 +14,50 @@ The next step is creating a backup policy and disaster recovery plan.
 ## Backup policy
 
 As part of your backup policy you should regularly create backups of UCP.
-To create a backup of UCP, use the `docker/ucp backup` command. This creates
-a tar archive with the contents of the [volumes used by UCP](../architecture.md)
-to persist data, and streams it to stdout.
 
-You need to run the backup command on a UCP manager node. Since UCP stores
-the same data on all manager nodes, you only need to create a backup of a
-single node.
+To create a UCP backup, you may use the `{{ page.docker_image }} backup` command
+against a single UCP manager, according to the instructions in the next section.
+This command creates a tar archive with the contents of all the [volumes used by
+UCP](../architecture.md) to persist data and streams it to stdout.
+
+You only need to run the backup command on a single UCP manager node. Since UCP
+stores the same data on all manager nodes, you do not need to capture periodic
+backups from more than one manager node.
 
 To create a consistent backup, the backup command temporarily stops the UCP
 containers running on the node where the backup is being performed. User
-containers and services are not affected by this.
+resources, such as services, containers and stacks are not affected by this
+operation and will continue operating as expected. Any long-lasting `exec`,
+`logs`, `events` or `attach` operations against the affected manager node will
+be disconnected.
 
-To have minimal impact on your business, you should:
+Additionally, if UCP is not configured for high availability, you will be
+temporarily unable to:
+* Log in to the UCP Web UI
+* Perform CLI operations using existing client bundles
 
-* Schedule the backup to take place outside business hours.
+To minimize the impact of the backup policy on your business, you should:
 * Configure UCP for high availability. This allows load-balancing user requests
-across multiple UCP controller nodes.
+across multiple UCP manager nodes.
+* Schedule the backup to take place outside business hours.
 
 ## Backup command
 
-The example below shows how to create a backup of a UCP controller node:
+The example below shows how to create a backup of a UCP manager node and
+verify its contents:
+
+```bash
+# Create a backup, encrypt it, and store it on /tmp/backup.tar
+$ docker run --rm -i --name ucp \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  {{ page.docker_image }} backup --interactive /tmp/backup.tar
+
+# Ensure the backup is a valid tar and list its contents
```

Comment:
Can we include a bit more information on what to visually search for. I'll see a list of files, but I'll have no idea whether they contain all the data or not.